src/timer/Timer.js

```diff
@@ -1,17 +1,18 @@
-function Timer(seconds) {
-  this.seconds = seconds;
-}
+class Timer {
+  constructor(seconds) {
+    this.seconds = seconds;
+  }
 
-Timer.prototype.start = function() {
-  var instance = this;
-  var timerInterval = setInterval(function() {
-    if (instance.seconds === 0) {
-      clearInterval(timerInterval);
-    }
+  start() {
+    this.timerInterval = setInterval(() => {
```

Comment:
Reserve this for persistent variables that are attached to the class. timerInterval is better defined using const.